Gifski/util.swift

```diff
@@ -938,11 +938,59 @@ extension NSView {
 		])
 	}
 
+	func centerX(inView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			centerXAnchor.constraint(equalTo: view.centerXAnchor)
+		])
+	}
+
+	func centerY(inView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			centerYAnchor.constraint(equalTo: view.centerYAnchor)
+		])
+	}
+
 	func addSubviewToCenter(_ view: NSView) {
 		addSubview(view)
 		view.center(inView: superview!)
 	}
 
+	func constraintTopBottom(bottomView view: NSView) {
```

Comment:
The method name here is confusing. It makes it sound like it's constraining something to the top **and** bottom of a view. I also don't think it's something that needs to be made an utility for as it's not a very common use-case, like compared to centering.

Same with the below.